UITests/Responses/TransferFunds/GetUser.json

```diff
@@ -0,0 +1,34 @@
+{
+    "token": "usr-token",
```

Comment:
You can reuse the UserIndividualResponse.json file in TransferMethod response folder